README.md

```diff
@@ -1,280 +1,53 @@
-
-%% ISA
-isaj --> isaa
-isax --> isaa
-isaj --> thoth
-
-isax --> fsspread
-
-%% ------ Tools ---------
-
-subgraph Tools
-    arcdotnet[/arcIO.NET\]
-    arcitect[[ARCitect]]
-    arccom[[arcCommander]]
-    swate[[Swate]]
-end 
-
-%% ----- Edges ------
-
-Tools --> ARCtrl
-
-arcdotnet --> fsspreadx
-arcdotnet --> systemio
-arcitect --> excelJS
-arcitect --> node
-arccom --> arcdotnet
-
-
-fsspreadx -.-> fsspread
-systemio -.-> thoth
-excelJS -.-> fsspread
-node -.-> thoth
-
-%% ----- Styling ------
-
-%% dotnet
-style Tools stroke:#3A0EC6,stroke-width:2px
-style fsspreadx fill:#8C71E3,stroke:#3A0EC6,stroke-width:2px,color: black
-style arcdotnet fill:#8C71E3,stroke:#3A0EC6,stroke-width:2px,color: black
-style arccom fill:#8C71E3,stroke:#3A0EC6,stroke-width:2px,color: black
-style systemio fill:#8C71E3,stroke:#3A0EC6,stroke-width:2px,color: black
-
-%% javascript
-style excelJS fill:#C1AD09,stroke:#EFD81D,stroke-width:2px,color: black
-style arcitect fill:#C1AD09,stroke:#EFD81D,stroke-width:2px,color: black
-style swate fill:#C1AD09,stroke:#EFD81D,stroke-width:2px,color: black
-style node fill:#C1AD09,stroke:#EFD81D,stroke-width:2px,color: black
-
-
-%% fable
-style ARCtrl stroke:#007B00,stroke-width:2px
-style ISA stroke:#007B00,stroke-width:2px
-style FileSystem stroke:#007B00,stroke-width:2px
-style CWL stroke:#007B00,stroke-width:2px
-style DataTransferObject stroke:#007B00,stroke-width:2px
+```fsharp
+#r "nuget: ARCtrl"
+``` 
 
-style fsspread fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style isax fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style isaa fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style isaj fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style arc fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style fs fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style cwl fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
-style thoth fill:#39B539,stroke:#007B00,stroke-width:2px,color: black
+```bash
+<PackageReference Include="ARCtrl" Version="1.0.0-alpha9" />
 ```
 
-## Stack
-
-```mermaid
-classDiagram 
-
-class arc["ARCtrl"] {
-    ISA
-    CWL
-    FileSystem
-}
-class isa["ISA"] {
-    Investigation
-    Studies
-    Assays
-    ArcTable
-    CompositeColumn
-    CompositeHeader
-    CompositeCell
-}
-class file ["FileSystem"] {
-    - FileSystemTree
-    - History
-}
-class cwl["CWL"] {
-    CommandlineTool?
-    Workflows
-}
-class io["IO"] {
-    FsSpreadsheet.ExcelIO
-    System.IO
-    exceljs
-    node fs
-}
-io <|--|> arc : Contracts
-arc <|--|> isa
-arc <|--|> cwl
-arc <|--|> file
+### JavaScript
 
+```bash
+npm i @nfdi4plants/arctrl
 ```
 
-# Libraries
+## Docs
 
-Mockups for each library are in the `ARC` folder. They contain API/Domain stubs to draft and test the new library designs.
+Currently we provide some documentation in form of markdown fiels in the `/docs` folder of this repository!
 
-## Design choices
+[Check it out!](/docs)
```

Comment:
Maybe directly include the links to the docs here?

Reply:
it is? 😅